includes/class-kirki-field.php

```diff
@@ -537,8 +537,8 @@ protected function set_tooltip() {
 		 */
 		protected function set_active_callback() {
 
-			if ( is_array( $this->active_callback ) && ! is_callable( $this->active_callback ) ) {
-				if ( isset( $this->active_callback[0] ) && is_array( $this->active_callback[0] ) ) {
+			if ( is_array( $this->active_callback ) ) {
+				if ( isset( $this->active_callback[0] ) ) {
 					$this->required = $this->active_callback;
```

Comment:
Why were these changed?

If for example now users use something like `'active_callback' => array( 'My_Class', 'my_function' ),` and have a class like this:
```php
class My_Class {
    public static function my_function( $something ) {
        // code
    }
}
```

it won't properly work...
@naxvog is there any particular reason we can't keep these?